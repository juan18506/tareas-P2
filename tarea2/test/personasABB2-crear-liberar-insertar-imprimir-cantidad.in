crearArbolPersonas

imprimirCantidadArbolPersonas

imprimirArbolPersonas

crearAgenda
crearPersona 5 55 Persona5
crearFecha 5/55/55
crearEvento 5 Descripcion5
agregarEventoPersona
insertarPersonaArbolPersonas

imprimirArbolPersonas

crearAgenda
crearPersona 10 100 Persona10
insertarPersonaArbolPersonas

imprimirCantidadArbolPersonas
imprimirArbolPersonas

crearAgenda
crearPersona 2 22 Persona2
insertarPersonaArbolPersonas

imprimirCantidadArbolPersonas
imprimirArbolPersonas

crearAgenda
crearPersona 20 200 Persona20
insertarPersonaArbolPersonas

crearAgenda
crearPersona 7 77 Persona7
insertarPersonaArbolPersonas

crearAgenda
crearPersona 1 11 Persona1
insertarPersonaArbolPersonas

crearAgenda
crearPersona 3 33 Persona3
insertarPersonaArbolPersonas

imprimirArbolPersonas

liberarArbolPersonas

Fin
