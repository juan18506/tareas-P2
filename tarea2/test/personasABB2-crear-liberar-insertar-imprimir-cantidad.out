1>2>La cantidad de personas en el árbol es: 0.
3>4>5>6>7>8>9>10>Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
11>12>13>14>La cantidad de personas en el árbol es: 2.
15>Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 10:  Persona10, 100 años
16>17>18>19>La cantidad de personas en el árbol es: 3.
20>Persona 2:  Persona2, 22 años
Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 10:  Persona10, 100 años
21>22>23>24>25>26>27>28>29>30>31>32>33>Persona 1:  Persona1, 11 años
Persona 2:  Persona2, 22 años
Persona 3:  Persona3, 33 años
Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
Persona 20:  Persona20, 200 años
34>35>Fin.
